From 4c0568d9420e662136209052feed9875282fe545 Mon Sep 17 00:00:00 2001
From: Felipe Balbi <felipe.balbi@linux.intel.com>
Date: Mon, 23 May 2016 14:08:47 +0300
Subject: [PATCH 20/41] usb: dwc3: gadget: remove udelay() from generic cmd

We want commands to finish ASAP, so let's remove
that udelay() call.

Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
(cherry picked from commit 88811f7b722c417ececbc7f278c26d8df8606d02)
Signed-off-by: Janusz Dziedzic <januszx.dziedzic@intel.com>

Change-Id: Iac2805b3b0687ffcde7cf054a8ae86ac991b42bd
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-34814
Reviewed-on: https://android.intel.com:443/519790
---
 drivers/usb/dwc3/gadget.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/dwc3/gadget.c b/drivers/usb/dwc3/gadget.c
index 28b3fc4a7..8e51e0d 100644
--- a/drivers/usb/dwc3/gadget.c
+++ b/drivers/usb/dwc3/gadget.c
@@ -296,8 +296,6 @@ int dwc3_send_gadget_generic_command(struct dwc3 *dwc, unsigned cmd, u32 param)
 				ret = -EINVAL;
 			break;
 		}
-
-		udelay(1);
 	} while (timeout--);
 
 	if (!timeout) {
-- 
1.9.1

